tests/test_queries.py

```diff
@@ -290,24 +291,24 @@ def test_get_stmts_for_pmid():
 def test_get_stmts_for_mesh_id_w_children():
     # Two queries:
     # 1. evidences for publications with pmid having mesh annotation
-    # 2. statements for the evidences in 2
+    # 2. statements for the evidences in 1
     client = _get_client()
     mesh_id = ("MESH", "D000068236")
     stmts = get_stmts_for_mesh(mesh_id, client=client)
     assert stmts
-    assert isinstance(stmts[0], Activation)
```

Comment:
why did this change?

Reply:
Not sure, I checked the statements from the query and the Inhibition is new since before as I remember it. Anyway, it's probably a bad idea to test something that's not static.